app.py

```diff
@@ -0,0 +1,89 @@
+from distutils.command.build import build
+import streamlit as st
+import pandas as pd
+import numpy as np
+import glob
+import datasets
+from datasets import load_dataset
+
+# data = load_dataset("biodatasets/<dataset_name>/<dataset_name>.py", name="<dataset_name>_bigbio_<schema>")
+# import os
+
+st.title('BigBIO datasets')
+
+# Add a selectbox to the sidebar:
+
+st.sidebar.header('Dataset Name')
+
+file_paths = glob.glob('./biodatasets/*')
+
+files = [i.split('/')[2] for i in file_paths]
+
+dataset_name = st.sidebar.selectbox(
+    'Name of dataset',
+    tuple(files)
+)
+
+schema_select = st.sidebar.selectbox("Schema", ("Source Schema", "BigBio Schema"))
+
+
+# This code snippet is from the datasets viewer to display the details in a clean way.
+def render_features(features):
+    if isinstance(features, dict):
+        return {k: render_features(v) for k, v in features.items()}
+    if isinstance(features, datasets.features.ClassLabel):
+        return features.names
+
+    if isinstance(features, datasets.features.Value):
+        return features.dtype
+
+    if isinstance(features, datasets.features.Sequence):
+        return {"[]": render_features(features.feature)}
+    return features
+
+
+def load_ds(dataset_name):
+    path = f"biodatasets/{dataset_name}/{dataset_name}.py"
+    module_path = datasets.load.prepare_module(path, dataset=True)
+    builder_cls = datasets.load.import_main_class(module_path[0], dataset=True)
```

Comment:
streamlit is so cool ... was trying this out and got an error b/c I have `datasets >= 2.0` ... seems they deprecated `prepare_module` ... this is the new way, 
```
path = f"biodatasets/{dataset_name}/{dataset_name}.py"
module = datasets.load.dataset_module_factory(path, dataset=True)
builder_cls = datasets.load.import_main_class(module.module_path, dataset=True)
```